code/script.js

```diff
@@ -1,51 +1,270 @@
+
+
+setTimeout ( () =>
+botReply ('First of all did you know that : The top of a pineapple, after cleaning and drying, can be planted in soil and a new plant will grow. '), 500)
+
+
+
+
+
+wrapper.innerHTML =`
+<button id="fun-fact" > Fun Fact </button>)
+<button id="history" > History </button>
+<button id="order" > Send me a pineapple!! </button>`
+
+//why has this eventlistneer started to show up as a bubble?
+document.getElementById('fun-fact').addEventListener('click', () => userReplyOne('fun'))
+
+
+document.getElementById('history').addEventListener('click', () => userReplyOne("history"))
+document.getElementById('order').addEventListener('click', () => answerThree("order"))
+
+}
+//answer and first interaction
+
+const userReplyOne = (chooseOne) => {  
+if (chooseOne === 'fun'){
+
+
+  setTimeout ( 
+    () =>
+  userReply ("yey funfact"), 1000)
+
+  setTimeout ( () =>
+  
+  botReply  ("Did you know that a pineapple is not an apple, or pine? It’s actually a berry!"), 1000)
+
+  setTimeout ( () =>
+  botReply ("Are you ready to order your pineapple? or do you need a little bit of history first?"), 1000)
+
+  wrapper.innerHTML =`
+
+  <button id="order" > Send me a pineapple!! </button>
+  <button id="historyOne" > A little bit of history on my side </button> 
+
+  `;
+  //Button history has to go on its own
+
+  //this one is separate at the end ifs so we dont get to many ifs
+  document.getElementById('historyOne').addEventListener('click', () => answerTwo('historyEnd'))
+  document.getElementById('order').addEventListener('click', () => answerThree( 'order'))
+
+}
+else if (chooseOne === "history") {
+
+  setTimeout ( () =>
+  userReply('so important to know the history'), 1500)
+
+  setTimeout ( () =>
+  botReply ("The pineapple plant is a tropical fruit that is indigenous to South America. Originally coming from the area between southern Brazil and Paraguay."), 1500)
+
+
+  setTimeout ( () =>
+  botReply ("Are you ready to order your pineapple? or mabey read some funfacts first?"), 1000),
+
+  wrapper.innerHTML = `
+
+
+<button id="fun-fact" > More fun facts </button>
+
+  <button id="order" > Send me a pineapple!! </button>`
+
+
+
+document.getElementById('fun-fact').addEventListener('click', () => answerTwo('funFactTwo')) //den går till en helt ny function med nya ifs!
+
+  document.getElementById('order').addEventListener('click', () => answerThree('order'))
+
+}}
+
+
+
+answerTwo = (type) => {
+  console.log ('working1')
+  if (type === 'historyEnd'){
+
+    setTimeout ( () =>
+    botReply(`More history for the people: The pineapple spread throughout South America, the Caribbean, Central America and Mexico, where it was cultivated by the Mayas and the Aztecs. Columbus came across the pineapple in 1493 and took it back to Europe.`), 1500)
+
+    setTimeout ( () =>
+    botReply ('Are you ready to order your pineapple?'), 1000)
```

Comment:
Same here with the time, the first botReply is set to 1500 and the second to 1000. This means the second "Are you ready to order your pinapple?" will show up before the history answe!